docs/DeveloperGuide.md

```diff
@@ -283,32 +283,66 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `EmployeeManager` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC1 - View all employees**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User requests to list all employees.
+1. EmployeeManager shows a list of employees.
+1. Use case ends.
 
-    Use case ends.
+**Use case: UC2 - Add a new employee**
 
-**Extensions**
+**MSS**
 
-* 2a. The list is empty.
+1. User requests to add a new employee.
+1. EmployeeManager requests for details of the employee to be added.
```

Comment:
Should this be condensed to one sentence? e.g. User sends employee data to be added to EmployeeManager